package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@saddlebackchurch/react-cm-ui",
-  "version": "10.7.5",
+  "version": "10.7.7",
```

Comment:
Please revert.  I do this as part of the release process.  Let's please keep this PR focused only on the icon changes.

Reply:
changed